### PR TITLE
added styles for all card variants, added one style for icon button

### PR DESCRIPTION
**Pull request by aishwarya-bm**
added styles for all card variants, added one style for icon button & few minor changes for button component for better reusability.
Card variants:
1. only text
2. card with dismiss
3. basic vertical card
4. card with badge
5. card with badge & wishlist icon(top right)
6. card with text overlay
7. card with header on top
8. only icons
9. only buttons
10. horizontal card

https://user-images.githubusercontent.com/66695068/152656839-bf976fd7-206e-4b37-b236-81e98149bc25.mp4

### CODE REVIEW

File: stylesheets/components/card.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-direction: column;
```
Comment (kjais1720) [inline stylesheets/components/card.css]: you can use utility class for the sam

**Review by kjais1720 — COMMENTED** [review]
Looks good, just the flex and flex direction could be replaced by a single utility class

File: stylesheets/components/card.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+/* card with a badge */
+.card-badge {
+  position: absolute;
+  top: 20px;
+  left: -2px;
+  background-color: var(--primary-color);
+  color: white;
+  font-weight: 600;
+  font-size: 0.75rem;
+  padding: 0.2rem 0.5rem;
```
Comment (vivekmehto) [inline stylesheets/components/card.css]: you can use badge class of badge component

**Review by vivekmehto — COMMENTED** [review]
Everything looks good just one suggestion you can use badge component.

File: stylesheets/components/card.css (hunk 1 above)
Comment (aishwarya-bm) [reply stylesheets/components/card.css]: made the changes, thanks!

File: stylesheets/components/card.css (hunk 2 above)
Comment (aishwarya-bm) [reply stylesheets/components/card.css]: extracted the style into badge

**Review by kjais1720 — APPROVED** [review]
All good👍

**Review by vivekmehto — APPROVED** [review]

**Pull request merged**